duckduckgo/Android
# Android: Product copy guide

Task/Issue URL: https://app.asana.com/0/1142021229838617/1198918663253946/f
Tech Design URL: 
CC: https://app.asana.com/0/488551667048375/1199208423932005/f

**Description**:
As per https://app.asana.com/0/72649045549333/1193747659422225/f, this PR brings to the app a big update on our copies based on a product copy review to ensure all our product follow the same copy guidelines. We have fixed some inconsistencies and followed recommendations from our copywriting team.


**Steps to test this PR**:
1. Run the app using different languages and ensure we haven't introduced any breakage.

- CTA translated: 
Follow steps in https://github.com/duckduckgo/Android/pull/1021 but using another non-english language


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
